src/Components/Carousel/index.js

```diff
@@ -1,36 +1,30 @@
 import React, { Component } from 'react';
 import './Carousel.css';
-import PopUp from '../PopUp';
+import GameCard from '../GameCard';
+import { uid } from 'react-uid';
 
```

Comment:
Any reason you went back to the uid?

Reply:
Yes, I used just the string from the game object and I was getting errors for having duplicate keys. I imported uid because it takes an object and provides a unique key even if we used the same object elsewhere(different carousel). We currently have no errors in the dev tools console using this method.